docquality.rst

```diff
@@ -88,7 +88,9 @@ installed.  The devguide HTML can be built by running::
     $ make html
 
 in the checkout directory, which will write the files to the ``_build/html``
-directory.
+directory.  Note that ``make check`` is automatically run when
+you submit a :doc:`pull request <pullrequest>`, so you should make
+sure that it runs without errors.
```

Comment:
This doesn't actually seem to be the case for the devguide repo, so either the sentence should be removed or the check should be added.